src/common/services/translator.ts

```diff
@@ -13,13 +13,16 @@ import * as esCatalog from "readium-desktop/resources/locales/es.json";
 import * as frCatalog from "readium-desktop/resources/locales/fr.json";
 import * as jaCatalog from "readium-desktop/resources/locales/ja.json";
 import * as nlCatalog from "readium-desktop/resources/locales/nl.json";
+import * as ptBrCatalog from "readium-desktop/resources/locales/pt-br.json";
+import * as ptPtCatalog from "readium-desktop/resources/locales/pt-pt.json";
 
 import * as deLang from "readium-desktop/resources/locale-names/deLang.json";
 import * as enLang from "readium-desktop/resources/locale-names/enLang.json";
 import * as esLang from "readium-desktop/resources/locale-names/esLang.json";
 import * as frLang from "readium-desktop/resources/locale-names/frLang.json";
 import * as jaLang from "readium-desktop/resources/locale-names/jaLang.json";
 import * as nlLang from "readium-desktop/resources/locale-names/nlLang.json";
+import * as ptLang from "readium-desktop/resources/locale-names/ptLang.json";
```

Comment:
The file ptLang.json needs to be duplicated for pt-pt and pt-br

Reply:
Changes made as suggested, 1 file created (pt-brLang.json) and one renamed (pt-ptLang.json). translator.ts changed to replace ptLang.json by both files. Thanks and waiting for any other change needed.